src/components/panels/TimePanel.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <HubPanelWrap :title="headertext">
+    <b-list-group>
+      <b-list-group-item :style="styleObject" class="list-group-item-borderless">
+        <span class="pull-left">
+          <b>Currently it is:</b>
+        </span>
+        <span v-if="time.isDay && location !=='Vallis'" class="pull-right day">Day</span>
+        <span v-else-if="location !=='Vallis' && !time.isDay" class="pull-right night">Night</span>
+        <span v-else-if="location =='Vallis' && !time.isWarm" class="pull-right cold">Cold</span>
+        <span v-else class="pull-right day">Warm</span>
+      </b-list-group-item>
+      <b-list-group-item :style="styleObject" class="list-group-item-borderless">
+        <span class="pull-left" v-if="location !=='Vallis'">
```

Comment:
In the future it'd be nice to split up Vallis and PoE a better way -- perhaps with subclasses or callback arguments, for example.

Reply:
go ahead and make an issue for this one